docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+Auth Service is running.
+
+## Prerequisites
+
+- Teleport Identity Governance enabled for your account. Optionally, Identity Security should
+also be enabled if you choose to enable Identity Security integration.
+- Your user must have privileged administrator permissions in the Azure account.
+- For the OIDC IdP setup, the Teleport cluster must be publicly accessible from the internet.
+
+
+## Guided Configuration
+
+The guided configuration expects setting up a new Entra ID auth connector and OIDC
+IdP integration as a default. 
+
+<Tabs groupId="entra-id">
+<TabItem label="Teleport as OIDC provider (Web UI)" >
+
+## Step 1/3: Generate configuration script
+In the Teleport Web UI, from the side-navigation, select “Add New > Integration”.
+
+In the integration UI, select the “Microsoft Entra ID” tile. 
+![Entra ID Integration](../../img/igs/entraid/entra-tile.png)
+
+In the Microsoft Entra ID configuration UI, you will notice a default integration name “entra-id”
+is already populated for you. You will need to select Teleport user(s) that will be assigned as 
+the default owner of Access Lists that are created for your Entra ID groups. 
+
+![Entra ID Integration](../../img/igs/entraid/entra-default-owners.png)
+
+In the next step, you will be provided with a Entra ID configuration script.
+
+## Step 2/3: Configure Entra ID
+
+(!docs/pages/includes/identity-governance/azure-shell.mdx!)
+
+## Step 3/3: Finish plugin installation
+
+Once the script is done setting up the necessary permissions,
+it prints out tenant ID and client ID. Copy these values and provide it to the
+Web UI to finish the integration.
+
+![Entra ID Integration](../../img/igs/entraid/entra-finish.png)
+
+</TabItem>
+
+<TabItem label="Teleport as OIDC provider (tctl)" >
+
+## Step 1/3: Generate configuration script (tctl)
+
+To begin integration, run the `tctl plugins install entraid` command.
+
+```code
+$ tctl plugins install entraid \
+    --name entra-id-default \
+    --auth-connector-name entra-id \
+    --default-owner=<Var name="Access List Owner"/> \
+    --default-owner=someOtherOwner@teleport.sh  \
+    --auth-server <Var name="example.teleport.sh:443" /> 
+
+Step 1: Run the Setup Script
+...
+
+Once the script completes, type 'continue' to proceed, 'exit' to quit: 
+```
+
+## Step 2/3: Configure Entra ID (tctl)
+
+(!docs/pages/includes/identity-governance/azure-shell.mdx!)
+
+## Step 3/3: Finish plugin installation (tctl)
+
+Once the script is done setting up the necessary permissions,
+it prints out tenant ID and client ID. Copy these values and provide it to the
+`tctl` to finish the integration.
+
+</TabItem>
+
+<TabItem label="System credential (tctl)" >
+
+## Step 1/4: Set up system credential
+
+You will need to grant Azure Identity with the necessary permissions required for the Entra ID
+integration.
+
+In the Azure Portal, find the identities linked to your Teleport Auth Service VMs,
```

Comment:
lets rephrase this with the comments I left before about kube pods and other vms